### DISCUSSION TITLE
Would cause issues if `conscript` is installed (solution found)

### DISCUSSION THREAD

**Discussion by zoonfafer** [scalameta/nvim-metals#142] [General]
Just noting that if you have [`conscript`](http://www.foundweekends.org/conscript/usage.html) installed, its executable (also called `cs`) would conflict with `coursier`'s `cs`.

Running `:MetalsInstall` would yield `conscript`'s usage message (inspected using `:echo metals#status()`).

Took some time to figure this out, hope it might help someone who reads this.

*Solution*: It seems [`conscript` is being considered for sunsetting](https://github.com/foundweekends/conscript/issues/164) (at the time of writing), so it should probably be fine to remove `conscript` from your `PATH` altogether.

**Comment by ckipp01** [comment]
Thanks for pointing this out @zoonfafer. I've actually _never_ heard of `conscript` before this. It's actual a bit crazy to see some of the concepts that are the same between the two. With that being said, this is super confusing, especially for newcomers. I've actually super surprised that I haven't had anyone hit on this before. If the plan is to sunset this, then there should be no issues. I'd imagine at a larger ecosystem level, it's not a good idea to have two different `cs` commands, so I hope this is done soon if it is the plan. There are tons of things in the Scala ecosystem that are now relying on coursier `cs`, so I'm curious if this is popping up more in other places as well.
